crypto/src/main/scala/org/bitcoins/crypto/SchnorrNonce.scala

```diff
@@ -14,7 +14,7 @@ case class SchnorrNonce(bytes: ByteVector) extends NetworkElement {
   }
 }
 
-object SchnorrNonce extends Factory[SchnorrNonce] {
+object SchnorrNonce extends Factory[SchnorrNonce] with CryptoTrait {
```

Comment:
Can these be reverted now?

Reply:
oh yeah, i reverted the entire core project (and it worked!) but i suppose it doesn't get the `crypto` module. I'll just delete `CryptoTrait` now as it is unnecessary i believe